locales/en/messages.json

```diff
@@ -3706,7 +3706,7 @@
         "message": "Dynamic Notch Filter"
     },
     "pidTuningDynamicNotchFilterHelp": {
-        "message": "Dynamic Notch Filter tracks peak motors noise frequency and places one or two notch filters with their center at that frequency."
+        "message": "Dynamic Notch Filter tracks peak gyro noise frequencies and places one to five notch filters with their center at these frequencies on every axis."
```

Comment:
What happens with the old version? The text is not correct anymore. Two options:
- Make two different messages.
- Simply use something more general, like `and places notch filters with` without specifying number.

Reply:
Good point @McGiverGim thanks for review it, fixed now.